### PR TITLE
2023 Q2 Hyperledger Firefly report

### LINKED ISSUE #1339: Reminder to submit the 2023 Q2 quarterly project report
Dear @hyperledger/firefly-committers, please submit the 2023 Q2 quarterly report (due 2023-06-08) for @hyperledger/toc review and mark this issue as complete.

### PR DESCRIPTION
**Pull request by aznrayizzle**

### CODE REVIEW

File: project-reports/2023/images/HLFF_ContstrQ22023.png.png
Comment (tkuhrt) [inline project-reports/2023/images/HLFF_ContstrQ22023.png.png:1]: Rename to a single `.png` extension. Currently does not show in the report.

**Review by jimthematrix — APPROVED** [review]

**Review by petermetz — APPROVED** [review]
LGTM

**Review by swcurran — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]
How can we help in improving PR/committer diversity?

**Review by mbrandenburger — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

File: project-reports/2023/images/HLFF_ContstrQ22023.png.png
Comment (tkuhrt) [reply project-reports/2023/images/HLFF_ContstrQ22023.png.png:1]: @aznrayizzle : if we can get this change done, we can merge this PR.

Comment (tkuhrt) [reply project-reports/2023/images/HLFF_ContstrQ22023.png.png:1]: @ryjones : Did this get done before the merge?

**Pull request merged**
